packages/portal/spec-renderer/src/components/operations-list/OperationsListItem.vue

```diff
@@ -94,8 +94,6 @@ const methodName = computed((): string => {
   }
 
   &:hover {
-    background: var(--kong-ui-spec-renderer-operations-list-item-background-hover, $kui-color-background-inverse);
```

Comment:
We need to add this back for theming purposes. Can we adjust the fallback variable (`$kui-color-background-inverse`) instead?

Reply:
Gotcha, adjusted fallback variable